pkg/crc/machine/generate_bundle.go

```diff
@@ -31,6 +31,11 @@ func (client *client) GenerateBundle() error {
 	// Stop the cluster
 	currentState, err := client.Stop()
 	if err != nil {
+		if forceStop {
```

Comment:
Should this branch off before we call `client.Stop()` user will still have to wait for 2mins even if `force` flag is used?

Reply:
@anjannath yes, this is same experience what user have for `stop` command where even using `-f` need to wait for 2 mins.